src/stories/layouts/Grid/styles.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Grid
+
+%shared {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-column-gap: rem(15px);
+}
+
+.grid {
+    @extend %shared;
+    @include bp-tablet {
+        grid-template-columns: repeat(2, 1fr);
+    }
+
+    @include bp-tablet-lg {
+        grid-template-columns: repeat(3, 1fr);
+    }
+
+    @include bp-desktop {
+        grid-template-columns: repeat(4, 1fr);
+        grid-column-gap: $spacing-3;
+    }
+}
+
+.grid--one {
+    @extend %shared;
+    grid-template-columns: repeat(1, 1fr);
+}
+
+.grid--two {
+    @extend %shared;
+    grid-template-columns: repeat(2, 1fr);
+}
+
+.grid--three {
+    @extend %shared;
+    grid-template-columns: repeat(3, 1fr);
+}
+
+.grid--four {
+    @extend %shared;
+    grid-template-columns: repeat(4, 1fr);
+}
+
+.grid__item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
```

Comment:
Remove this. We don't want to make assumptions on the display of the grid items.